packages/app/src/router/index.ts

```diff
@@ -16,6 +16,12 @@ const routes: RouteRecordRaw[] = [
     component: () => import('../views/Welcome.vue'),
     meta: { authentication: 'guests-only' },
   },
+  {
+    path: '/tag-details/:id',
```

Comment:
```suggestion
    path: '/tag/:id',
```

I would remove hypfen from the uri.

Reply:
good idea